library-best-practices/JLBP-16.md

```diff
@@ -0,0 +1,28 @@
+[JLBP-16] Produce a BOM for multi-module projects
+-------------------------------------------------
+
+### The BOM (Bill of Materials)
+
+- A BOM is a `pom.xml` file that has a `<dependencyManagement>` section that lists all the modules of a project. It does not have `<modules>` or `<dependencies>` elements itself. It also has `<type>pom</type>` declared.
```

Comment:
It can have a modules element.
The <dependencyManagement> declares dependencies, no modules.

Reply:
Do you have an example of a BOM with a modules element?

Reply:
The first BOM shown on this page has a modules element:

https://maven.apache.org/guides/introduction/introduction-to-dependency-mechanism.html 

It's right after "The root of the project is the BOM pom. It defines the versions of all the artifacts that will be created in the library. Other projects that wish to use the library should import this pom into the dependencyManagement section of their pom."